Corona-Warn-App/src/main/res/values-de/vaccination_strings.xml

```diff
@@ -37,22 +37,21 @@
 
     <!-- XTXT: Vaccination List title-->
     <string name="vaccination_list_title">Digitaler Impfnachweis</string>
-    <!-- XTXT: Vaccination List complete vaccination subtitle-->
-    <string name="vaccination_list_complete_vaccination_subtitle">SARS-CoV-2-Impfschutz</string>
-    <!-- XTXT: Vaccination List top card title-->
-    <string name="vaccination_list_top_card_title">SARS-CoV-2 Impfung</string>
     <!-- XTXT: Vaccination List top card subtitle-->
-    <string name="vaccination_list_top_card_subtitle">Unvollständiger Impfschutz</string>
-    <!-- XTXT: Vaccination List name card subtitle-->
     <string name="vaccination_list_name_card_subtitle">geboren %1$s</string>
     <!-- XTXT: Vaccination List top card subtitle-->
     <string name="vaccination_list_vaccination_card_title">Impfung %1$s von %2$s</string>
     <!-- XTXT: Vaccination List top card subtitle-->
     <string name="vaccination_list_vaccination_card_subtitle">durchgeführt am %1$s</string>
-    <!-- XTXT: Vaccination List top card subtitle-->
-    <string name="vaccination_list_certificate_card_title">COVID-19-Prüfzertifikat</string>
-    <!-- XTXT: Vaccination List top card subtitle-->
-    <string name="vaccination_list_certificate_card_subtitle">Noch %1$d Tage gültig</string>
+    <!-- XTXT: Vaccination List immunity information card body-->
+    <plurals name="vaccination_list_immunity_card_body">
+        <item quantity="one">Sie haben nun alle Impfungen erhalten. Allerdings ist der Impfschutz erst in %1$d Tag vollständig.</item>
+        <item quantity="other">Sie haben nun alle Impfungen erhalten. Allerdings ist der Impfschutz erst in %1$d Tagen vollständig.</item>
```

Comment:
Can't we keep only `one` and `other`? should not be enough?

Reply:
Yes, should be enough. However, as far as I can remember from the last time we used quantity strings, UA said the translation tool needs to have all of those.